dd-java-agent/agent-builder/src/main/java/datadog/trace/agent/tooling/CombiningTransformerBuilder.java

```diff
@@ -130,7 +130,10 @@ private void prepareInstrumentation(InstrumenterModule module, int instrumentati
     }
     helperTransformer =
         helperClassNames.length > 0
-            ? new HelperTransformer(module.getClass().getSimpleName(), helperClassNames)
+            ? new HelperTransformer(
+                module.injectHelperClassesWithAgentCodeSource(),
```

Comment:
I think we can simplify this method name to just:
```suggestion
                module.useAgentCodeSource(),
```
and use javadoc to explain how/where it is used.

Reply:
Renamed the method